packages/playground/src/App.js

```diff
@@ -21,8 +21,19 @@ function App() {
   const REDUX_STORE_KEY = 'REDUX_STORE';
 
   // fluentui settings
-  const stackTokens = { childrenGap: 10 };
-
+  const stackTokens = { childrenGap: 0 };
+
+  // TODO: Add action buttons in tooltip later
+  const wordBreakTooltipProps = {
```

Comment:
it is not clear what is this WordBreakToolTip do could we add  2 liners  what is this doing/testing?

Reply:
this is the renderer for that tooltip, since it needs jsx. Not sure what you mean by 2 liners?